parts/k8s/containeraddons/kubernetesmasteraddons-antrea-daemonset.yaml

```diff
@@ -0,0 +1,495 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Actually this yaml creates a Deployment, a DaemonSet, and multiple other resources. Should we name it antrea-deployment.yaml instead?

Reply:
Most plugin use this convention. I named it accordingly.

Reply:
I updated it to just `"antrea.yaml"`. The convention you observed is a poor one, unfortunately, but totally understood why you were following the pre-existing patterns!